lib/oli_web/components/delivery/nav_sidebar.ex

```diff
@@ -19,7 +19,7 @@ defmodule OliWeb.Components.Delivery.NavSidebar do
 
   def main_with_nav(assigns) do
     ~H"""
-      <main role="main" class="h-screen flex flex-col relative lg:flex-row z-0">
+      <main role="main" class="flex flex-col relative lg:flex-row z-0">
```

Comment:
Removing h-screen was necessary to get popups to render in the correct location, I think it was a leftover mistake style here anyways